recipe/meta.yaml

```
@@ -1,37 +1,65 @@
-{% set name = "scipy-stubs" %}
 {% set version = "1.14.1.5" %}
+{% set scipy_version = version.rsplit('.', 1)[0] %}
 {% set python_min = "3.10" %}
 
 package:
-  name: {{ name|lower }}
+  name: scipy-stubs-split
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/scipy_stubs-{{ version }}.tar.gz
-  sha256: 66c160c6b72b1406edc7827879ef25ea81dc3d95ab7e4af3a2de836f27c26ce6
+#   url: https://pypi.org/packages/source/s/scipy-stubs/scipy_stubs-{{ version }}.tar.gz
+#   sha256: 66c160c6b72b1406edc7827879ef25ea81dc3d95ab7e4af3a2de836f27c26ce6
+  git_url: https://github.com/jorenham/scipy-stubs.git
```

This is not what I meant by using the tag.
```suggestion
  url: https://github.com/jorenham/scipy-stubs/archive/refs/tags/v{{ version }}.tar.gz
  sha256: ...
```